src/test/unit/keysim-globalifier.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+if (
+    jQuery.browser.name === "msie" &&
+    jQuery.browser.versionNumber <= 8
+   ) {
+    // This browser does not properly implement
+    // `Object.prototype.defineProperty`.
+    // This is required by the external keyboard event simulation module,
+    // `keysim`.
+    window.skipKeyboardShortcutTests = true;
+} else {
+    window.Keysim = require("keysim");
+    window.skipKeyboardShortcutTests = false;
+}
```

Comment:
I would be OK with creating a separate issue to add IE8 testing support and merging this in. Doesn't make sense to block getting user feedback at this point.